### PR TITLE
fix the reduce api en doc 

### PR DESCRIPTION
**Pull request by wangchaochaohu**
1.reduce_sum
image

2.reduce_mean
image

3reduce_max
image

4reduce_min
image

5reduce_prob
image

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -5035,21 +5035,27 @@ def reduce_sum(input, dim=None, keep_dim=False, name=None):
...
-                       will be named automatically.
+            than the :attr:`input` unless :attr:`keep_dim` is true, default
+            value is False.
+        name(str, optional): The default value is None.  Normally there is no need for
+            user to set this property.  For more information, please refer to :ref:`api_guide_Name`
 
     Returns:
-        Variable: The reduced Tensor variable.
+        Variable: Tensor, which performs sum operation on the specified dim of input tensor,
+        has the same data type with input data type.
```
Comment (cyj1986) [inline python/paddle/fluid/layers/nn.py]: Tensor, Results of summation  on the specified dim of input tensor, its data type is the same as input's.

Comment (wangchaochaohu) [reply python/paddle/fluid/layers/nn.py]: Done

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -5784,21 +5784,27 @@ def reduce_sum(input, dim=None, keep_dim=False, name=None):
...
-        dim (list|int|None): The dimensions along which the sum is performed. If
+        input (Variable): The input variable which is a Tensor, the data type is float32,
+            float64, int32, int64.
+        dim (list|int, optional): The dimensions along which the sum is performed. If
             :attr:`None`, sum all elements of :attr:`input` and return a
             Tensor variable with a single element, otherwise must be in the
             range :math:`[-rank(input), rank(input))`. If :math:`dim[i] < 0`,
             the dimension to reduce is :math:`rank + dim[i]`.
-        keep_dim (bool|False): Whether to reserve the reduced dimension in the
+        keep_dim (bool): Whether to reserve the reduced dimension in the
```
Comment (kuke) [inline python/paddle/fluid/layers/nn.py]: keep_dim(bool, optional)

**Review by cyj1986 — DISMISSED** [review]
LGTM

File: python/paddle/fluid/layers/nn.py (hunk 2 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/nn.py]: Done

**Review by cyj1986 — APPROVED** [review]
LGTM

**Review by heavengate — APPROVED** [review]

**Review by hong19860320 — APPROVED** [review]
LGTM

**Pull request merged**